source-available-recommendations.md

```diff
@@ -61,9 +62,71 @@ This is unlikely to be tenable as a long-term solution, particularly as
 the prior release is presumably unlikely to continue receiving maintenance
 updates, security fixes, etc.
 
-In either case, while a project's maintainers may request a license exception from the CNCF Governing Board, it is _extremely_ unlikely that one will be granted for the use of a dependency under a source available, non-open source license.
+### Fork and maintain
+
+Finally, the project might decide to take on the responsibility for maintaining its own fork of the open source version of the component, typically based on the final release prior to the change to a source available license.
+
+Please see [below](#considerations-for-forking-and-maintaining) for details to consider if the community desires to take this approach.
+
+### Exception requests
+
+In any case, while a project's maintainers may request a license exception from the CNCF Governing Board, it is _extremely_ unlikely that one will be granted for the use of a dependency under a source available, non-open source license.
+
+## Considerations for Forking and Maintaining
+
+A decision to fork and maintain a pre-existing independent project is typically not a small undertaking. Below are some of the considerations that a CNCF project would likely want to take into account when going this route. In any case, please reach out early to CNCF staff via the [CNCF Service Desk] to discuss details if you are considering this approach.
+
+### Governance and IP policy formalities
+
+The forked project’s ongoing maintenance could potentially be managed (1) within CNCF; (2) within a [separate open collaboration project under the Linux Foundation](https://www.linuxfoundation.org/projects); or (3) as part of a fully-separate community effort.
+
+If the fork would be part of the existing CNCF project which uses it, then it would need to comply with the IP policy from the [CNCF Charter]. Similarly, if it would be set up as its own CNCF project, it would also be subject to CNCF’s standard processes for onboarding new projects.
```

Comment:
```suggestion
If the fork would be part of the existing CNCF project which uses it, then it would need to comply with the IP policy from the [CNCF Charter] and be recorded as a sub-project subject to the CNCF project's existing governance processes. Similarly, if it would be set up as its own CNCF project, it would also be subject to CNCF’s standard processes for onboarding new projects; [initiating an application to join the CNCF](https://github.com/cncf/toc/tree/main/process).
```